internal/ui/src/components/empty.tsx

```diff
@@ -0,0 +1,72 @@
+import type * as React from "react";
+
+import { cn } from "../lib/utils";
+
+interface EmptyRootProps extends React.HTMLAttributes<HTMLDivElement> {
+  fill: boolean | undefined;
+}
+export function Empty({ className, children, ...props }: EmptyRootProps) {
+  return (
+    <div
+      className={cn(
+        props.fill ? "h-full w-full" : "",
+        "animate-in fade-in-50 flex-col p-8 text-center justify-center",
```

Comment:
there's a flex-col and justify-center here but no flex, they likely don't do anything